Tests/test_identify_kinematic_chain.m

```diff
@@ -0,0 +1,22 @@
+
+clear
+clc
+
+load('C:\GBW_MyPrograms\PredSim_test\Tests\Falisse_et_al_2022_Results\Falisse_et_al_2022_v1.mat')
+
+S = R.S;
+osim_path = model_info.osim_path;
+
+S.subject.base_joints_legs = 'hip';
+S.subject.base_joints_arms = {'acromial_r'};
```

Comment:
can we define {'hip_r','hip_l'} to S.subject.base_joints_legs. Need more description in readMe.

Reply:
yes, I added some extra info to the settings in the readme